heron/executor/src/python/heron_executor.py

```diff
@@ -621,6 +621,7 @@ def _get_streaming_processes(self):
         ','.join(map(lambda x: x[0], instance_info)),
         self.master_host,
         self.master_port,
+        self.tmaster_controller_port,
```

Comment:
is it `local_data_port`?

Reply:
in the executor, the ports are called master_port and tmaster_controller_port. They are labelled that way for both the tmaster container, as well as the regular container.
What we are doing is that using the tmaster_controller_port to be the local data port.